packages/create-react-admin/templates/ra-data-fakerest/src/App.spec.tsx

```diff
@@ -0,0 +1,36 @@
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
```

Comment:
Forgot to mention that on #9578 , but this file should probably be deleted since we now generate the tests dynamically

Reply:
Did you see this comment @djhi ?